src/hardware/input/mouseif_ps2_bios.cpp

```diff
@@ -926,7 +986,10 @@ void MOUSEBIOS_Subfunction_C2() // INT 15h, AH = 0xc2
 		set_return_value(BiosRetVal::Success);
 		break;
 	case 0x05: // initialize
-		if (reg_bh == 3 || reg_bh == 4) {
+		if (reg_bh == 1 || reg_bh == 3 || reg_bh == 4) {
```

Comment:
Would it be better to extract the supported frame sizes of 1, 3, and 4 to constants?

Reply:
I don't think constants would improve readability - but I have extracted the check to a dedicated function, with comments explaining why we support each and every one of the frame sizes.